readme.md

```diff
@@ -1,5 +1,113 @@
-# .NET Foundation Project Application and Support
 
-This repo is used to manage .NET Foundation project applications. More information is available at in the [project application checklist](https://github.com/dotnet/foundation/blob/master/guidance/new-projects.md).
+# .NET Foundation Project Committee
 
-This repo is also used to provide support for existing projects.
+## Background
+
+The .NET Foundation, an independent 501(C)(6) corporation, is chartered to foster open development and collaboration around the growing collection of open source technologies for .NET. It serves as a forum for commercial and community developers alike to strengthen the future of the .NET ecosystem by promoting openness, community participation and rapid innovation.
+
+The mission of the .NET Foundation is to foster a community of .NET developers focused on collaborating on commercially friendly open source software projects that enhance the .NET platform and create value within the .NET ecosystem.
```

Comment:
Should we add something about supporting smaller projects too so it doesn't sound like we just want only corporates to use the software. Eg commerical and hobbyist friendly.

Reply:
The mission statement was copied from the original Advisory Council charter document.

Reply:
Doesn't mean it can't be more inclusive. That's the point of pull requests.

Reply:
Agree. I just wanted to clarify the origin of the mission statement included in this document.

Reply:
Resolving due to scope. The mission statement of the foundation can be addressed elsewhere.